en/ros2/px4_sdk.md

```diff
@@ -0,0 +1,336 @@
+# PX4 SDK
+
+The [PX4 SDK](https://github.com/PX4/px4_sdk) is a C++ library to simplify controlling PX4 from ROS 2.
+It allows to write external modes that are dynamically registered with PX4 and behave the same way as internal ones.
+A mode can send different types of setpoints, ranging from high-level navigation tasks all the way down to direct actuator controls.
+
+## Overview
+
+This diagram provides a conceptual overview:
+
+![ROS2 modes overview diagram](../../assets/middleware/ros2/px4_sdk/ros2_modes_overview.svg)
```

Comment:
The diagram is great. Say you're in payload delivery mode. Does PX4 have the concept that it is in a payload delivery mode, and if so is this worth showing? Or is it more that PX4 thinks it is under external control/some special mode.

It might not matter if the story is "ROS2 modes are first-class modes ". But you could put a dashed box to show that it has some presence in PX4. Just a thought.